FirebaseABTesting.podspec

```diff
@@ -35,20 +35,19 @@ Firebase Cloud Messaging and Firebase Remote Config in your app.
    'FirebaseCore/Sources/Private/*.h',
   ]
   s.requires_arc = base_dir + '*.m'
-  s.public_header_files = base_dir + 'Public/*.h', base_dir + 'Protos/developers/mobile/abt/proto/*.h'
-  s.private_header_files = base_dir + 'Protos/developers/mobile/abt/proto/*.h'
+  s.public_header_files = base_dir + 'Public/*.h'
```

Comment:
It's weird, but the Private headers should be in the `public_header_files` line additionally.  Full details in the podspec docs.

Reply:
Fixed in 8e836ca.